lib/events/auditlog_test.go

```diff
@@ -165,10 +165,14 @@ func (a *AuditTestSuite) TestSessionRecordingOff(c *check.C) {
 	c.Assert(found[0].GetString(EventLogin), check.Equals, "doggy")
 	c.Assert(found[1].GetString(EventLogin), check.Equals, "doggy")
 
-	// inspect the session log for "200". it should be empty.
+	// inspect the session log for "200". we should have two
```

Comment:
let's not use `we` in the code and instead refer to the part of the code or algo in question. When I read "we" it takes me time to process who is meant by "we"